.gitignore

```diff
@@ -0,0 +1,129 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+ # C extensions
+*.so
+
+ # Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+ # PyInstaller
+#  Usually these files are written by a python script from a template
+#  before PyInstaller builds the exe, so as to inject date/other infos into it.
+*.manifest
+*.spec
+
+ # Installer logs
+pip-log.txt
+pip-delete-this-directory.txt
+
+ # Unit test / coverage reports
+htmlcov/
+.tox/
+.nox/
+.coverage
+.coverage.*
+.cache
+nosetests.xml
+coverage.xml
+*.cover
+.hypothesis/
+.pytest_cache/
+
+ # Translations
+*.mo
+*.pot
+
+ # Django stuff:
+*.log
+local_settings.py
+db.sqlite3
```

Comment:
Please remove whatever is not required, for example db.sqlite3, Transactions etc, which are not used in our project. It is goof to keep what is required for our project.

Reply:
I have rebase my work on the latest master, and remote this .gitignore, see 25f67189141a4d943436e9c2c03b843981766aeb @Ram-srini